### DISCUSSION TITLE
NAS/TrueNas vs Harvesters

### DISCUSSION THREAD

**Discussion by Kruger1981** [madMAx43v3r/chia-gigahorse#212] [General]
I have a few HP Microservers that can accomodate 5HDDs each.
I am wondering what would be better (or if there is any difference at all), would it be better performance and power efficiency wise to have each of the micro servers set up as a NAS and have the farmer to read the plots via network or, set the micro servers as harvesters running windows or Linux?

**Comment by hajes** [comment]
You will never get power efficiency with multi-server setup.

Each HBA/expander uses perhaps 6W and gives you at least 8 drives. HBA + expander >20 drives.

NAS will most likely use more. my farm has got 64 drives @ 420W or 6.56W per drive...if I had just 20TB drives, you would have 1280TB or 0.33W/TB

**Reply by Kruger1981** [reply]
thank you for your input.
Yeah I had 12 external USB drives (a mix of 14tb, 16tb and 18tb) connected to a AC/DC PSU, but was moving a few things around and ended up braking the PSU so due to lack of power sockets placed the drives in 3 of those HP micro servers and power consumption has pretty much doubled.
Will need to get another PSU
